src/SFA.DAS.ProviderCommitments/SFA.DAS.ProviderCommitments.Web/Controllers/CohortController.cs

```diff
@@ -277,10 +277,17 @@ public async Task<IActionResult> ConfirmEmployer(ConfirmEmployerRequest request)
         [HttpPost]
         [Route("add/confirm-employer")]
         [Authorize(Policy = nameof(PolicyNames.HasContributorOrAbovePermission))]
-        public IActionResult ConfirmEmployer(ConfirmEmployerViewModel viewModel)
+        public async Task<IActionResult> ConfirmEmployer(ConfirmEmployerViewModel viewModel)
         {
             if (viewModel.Confirm.Value)
             {
+                var model = await _modelMapper.Map<HasDeclaredStandardsViewModel>(viewModel);
```

Comment:
You might to rename the `HasDeclaredStandardsViewModel` to something that expresses its intent a little more clearly (it's not really a vm). I would go for something like `ConfirmEmployerRedirectModel`, since it's a model that helps us decide where to redirect to from the Confirm Employer post.